test/test_triplets_classifiers.py

```diff
@@ -56,7 +62,7 @@ def test_no_zero_prediction(estimator, build_dataset):
     assert_array_equal(X[1], x)
   with pytest.raises(AssertionError):
     assert_array_equal(X[1], y)
-  # Assert the distance is the same for both
+  # Assert the distance is the same for both -> Wont work for b. similarity
```

Comment:
b. --> bilinear to be more explicit

Reply:
But anyway, is it so important to test this distance? (not really the purpose of the test) Because otherwise it looks like this test would pass also for bilinear learners